ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -146,6 +147,27 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Default:     true,
 				Description: "Indicates if all volumes attached to the server must reside in the same storage pool",
 			},
+			Arg_DeploymentTarget: {
+				Description: "The deployment of a dedicated host",
```

Comment:
```suggestion
				Description: "The deployment of a dedicated host. Max Items: 1.",
```

Reply:
That is not needed since `MaxItems` is define here.